packages/flutter/lib/src/material/expansion_tile.dart

```diff
@@ -669,14 +669,14 @@ class _ExpansionTileState extends State<ExpansionTile> with SingleTickerProvider
   }
 
   Widget? _buildLeadingIcon(BuildContext context) {
-    if (_effectiveAffinity(widget.controlAffinity) != ListTileControlAffinity.leading) {
+    if (_effectiveAffinity(widget.controlAffinity ?? ListTileTheme.of(context).controlAffinity) != ListTileControlAffinity.leading) {
       return null;
     }
     return _buildIcon(context);
   }
 
   Widget? _buildTrailingIcon(BuildContext context) {
-    if (_effectiveAffinity(widget.controlAffinity) != ListTileControlAffinity.trailing) {
+    if (_effectiveAffinity(widget.controlAffinity ?? ListTileTheme.of(context).controlAffinity) != ListTileControlAffinity.trailing) {
```

Comment:
See previous comment.

Reply:
resolved, see [commit](https://github.com/flutter/flutter/pull/146630/commits/eb31a043f8b4031cc63b3632221d98141c7aad3e)

Reply:
@TahaTesser I guess you should look at this?